NEWS.md

```diff
@@ -1,12 +1,13 @@
-# ggExtra 0.7.1
+# ggExtra 0.8
```

Comment:
I'd only change the version number to 0.8 when everything is ready for release

Reply:
OK I've changed this